src/locales/CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Joystream-org i18n contribution guide
+
+Firstly, we'd like to thank you for your interest in contributing to our internationalization project. Below you will be able to find a step-by-step explanation to help guide you through the process but also an overview of the necessary steps for the more advanced visitors to just skim through. If you have any questions or concerns, you are free to open an issue in [this repo](https://github.com/Joystream/joystream-org/issues) or get some help in [our discord community](https://discord.com/invite/DE9UN3YpRP).
+
+*Even if you don't plan on adding any changes yourself, you are more than welcome to check out any active Pull Requests and join the discussion by recommending a different translation or giving any insight towards making the translations better.*
+
+## Table of Contents
+
+- [File Structure and Translation Process](#file-structure-and-translation-process)
+- Contribution Process
+  - [Option 1 (making the changes on your local machine and pushing the changes to GitHub)](#option-1)
+  - [Option 2 (making the changes on GitHub's UI)](#option-2)
+  - [Option 3 (download file locally and send changes through Discord)](#option-3)
+- [Seeing the changes](#seeing-the-changes)
+
+## File Structure and Translation Process 
+
+The translations are written in .json files and here is an example excerpt from one of them: `(/en/sumer.json)`
+
+```js
+{
+  "sumer": {
+    "siteMetadata": {
+      "description": "Explore the Sumer testnet"
+    },
+    "hero": {
+      "title": "Sumer Network",
+      "text": "Among other changes, the Sumer testnet release offers major improvements to content uploads through our playback app."
+    },
+  ...
+  }
+}
+```
+To translate this file, you would need to:
+- Find the folder in `/src/locales` that corresponds to the language you want to translate to
+- Find a file in that folder (ex. `/src/locales/bs`) that has not yet been translated to that language (**check if there aren't any active PR's addressing the file as well**)
```

Comment:
I think this will be quite confusing for many users. Consider adding that they can also create new folders, and how to proceed:
> - Find a file in that folder (ex. `/src/locales/bs`) that has not yet been translated to that language (**check if there aren't any active PR's addressing the file as well**)

The folders are named by the two letter [ISO language code](https://www.loc.gov/standards/iso639-2/php/code_list.php), eg. `en` means English. If you want to translate to Bosnian, `bs`, check if there is already a folder with that name. If not, copy the `en` folder, and rename it `bs`. Then create a new folder, and name it `todo`, and place all the `*.json` files in it. Once you have completed translating one file, move it back to `bs`.

Reply:
This is along the lines of what I initially had but to enable this and enable users to be able to see the changes they've made I need to add the language to the Gatsby config that deals with the translation. Shouldn't be too difficult but it doesn't quite allow people to arbitrarily add new translations wherever they like.

That being said I really don't have a preference for either and we should be able to add a bunch languages to the config. But even then I would have to create the associated locales folders myself and fill them with the english translations as only adding the language to the config would make Gatsby create the related pages (e.g. `${baseUrl}/ru`) but instead of words it would have the keys displayed (e.g. `foundingMembers.landing.hero.text`) as there would be no values corresponding to those keys. I'm not sure that is something we want to have on the live page (it would be hidden though and you could only access it if you were really looking for it)?

With the current approach we can have one/multiple language(s) at a time as a  "community project" and make sure that it's polished and works.